### ISSUE TITLE
Missing files

### ISSUE THREAD

**Issue by ajeetraina** [loopDelicious/docker-extension#1]
I tried to clone and build the extension, but it threw the following error:

```
 make build
docker build --tag=joycelin79/newman-extension:latest .
[+] Building 1.3s (13/17)                                                                                                                                                  
 => [internal] load build definition from Dockerfile                                                                                                                  0.0s
 => => transferring dockerfile: 37B                                                                                                                                   0.0s
 => [internal] load .dockerignore                                                                                                                                     0.0s
 => => transferring context: 34B                                                                                                                                      0.0s
 => [internal] load metadata for docker.io/library/node:17.7-alpine3.14                                                                                               1.2s
 => [internal] load build context                                                                                                                                     0.0s
 => => transferring context: 976B                                                                                                                                     0.0s
 => CANCELED [client-builder  1/14] FROM docker.io/library/node:17.7-alpine3.14@sha256:539e64749f7dc6c578d744d879fd0ec37f3afe552ae4aca9744cc85121728c4c               0.0s
 => => resolve docker.io/library/node:17.7-alpine3.14@sha256:539e64749f7dc6c578d744d879fd0ec37f3afe552ae4aca9744cc85121728c4c                                         0.0s
 => => sha256:ce3358b66925f144108f66802a25e737a5ac991bf77c27bfd14dddb92d389371 1.16kB / 1.16kB                                                                        0.0s
 => => sha256:ed51fad11ee87160e0d69d19e6a6be7420b4965a516f57f5a713b4ebd634bc33 6.59kB / 6.59kB                                                                        0.0s
 => => sha256:539e64749f7dc6c578d744d879fd0ec37f3afe552ae4aca9744cc85121728c4c 1.43kB / 1.43kB                                                                        0.0s
 => CACHED [client-builder  2/14] WORKDIR /app/client                                                                                                                 0.0s
 => CACHED [client-builder  3/14] COPY client/package.json /app/client/package.json                                                                                   0.0s
 => CACHED [client-builder  4/14] RUN npm config set cache-folder /usr/local/share/.cache/npm-arm64                                                                   0.0s
 => CACHED [client-builder  5/14] RUN --mount=type=cache,target=/usr/local/share/.cache/npm-arm64                                                                     0.0s
 => CACHED [client-builder  6/14] RUN npm install --production                                                                                                        0.0s
 => CACHED [client-builder  7/14] COPY client /app/client                                                                                                             0.0s
 => CACHED [client-builder  8/14] RUN --mount=type=cache,target=/usr/local/share/.cache/npm-arm64     npm run build                                                   0.0s
 => ERROR [client-builder  9/14] COPY client/src/file.hbs /tmp/file.hbs                                                                                               0.0s
------
 > [client-builder  9/14] COPY client/src/file.hbs /tmp/file.hbs:
------
failed to compute cache key: "/client/src/file.hbs" not found: not found
make: *** [build] Error 1

```

**Comment by ajeetraina** [comment]
If I remove the file.hbs, it works perfectly well.

**Comment by benja-M-1** [comment]
I think this is now solved :)

**Comment by collabnix** [comment]
Can't wait to test this out.

**Comment by loopDelicious** [comment]
Resolved by @benja-M-1  in #3  👑
